styles.css

```diff
@@ -451,6 +475,78 @@ header {
   margin-left: 125px;
 }
 
+form {
+  margin-top: 43px;
+  width: 375px;
+  padding: 1em;
+  margin-left: 24px;
+}
+
+input,
+textarea {
+  font: 1em sans-serif;
+  width: 300px;
+  box-sizing: border-box;
+  border: 1px solid #999;
```

Comment:
Great job on adding form it works as intended but there are a few suggestions I think will make our form looks a little nicer 😇 (following the Figma design), As you can see the difference in the given image between the Figma design and our implementation there are few things I will list them here, let's kindly implement it.
I went a head and investigated the value from [the Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
to help you solve any confusion 🤓 

- [ ] Notice `inputs` do have a border at the bottom correct but with a different color they have `#DBD8D7` 
- [ ] Notice both `inputs` and `textarea` has 
```css
padding-inline: 12px;
```
- [ ] Notice `textarea` has background with slightly off white color and has no borders
```css
background-color:  #FBF8F7;
border: none
```

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/160646044-f70f01b7-f867-4815-9e12-db54ea814e53.png) | ![image](https://user-images.githubusercontent.com/87689547/160646114-c804cab0-ad2f-4f41-a127-505f07be99f0.png)